backend/app/api/endpoints/knowledge.py

```
@@ -488,6 +490,48 @@ def update_knowledge_base_type(
         )
 
 
+@router.post(
+    "/{knowledge_base_id}/migrate",
+    response_model=KnowledgeBaseMigrateResponse,
+)
+@trace_sync("migrate_knowledge_base_to_group", "knowledge.api")
+def migrate_knowledge_base_to_group(
+    knowledge_base_id: int,
+    data: KnowledgeBaseMigrateRequest,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)